gson/src/test/java/com/google/gson/InnerClassExclusionStrategyTest.java

```diff
@@ -26,8 +26,8 @@
  * @author Joel Leitch
  */
 public class InnerClassExclusionStrategyTest extends TestCase {
-  public InnerClass innerClass = new InnerClass();
```

Comment:
I'm not sure this change is great. Previously the test showed that this field was excluded, _despite_ being public. Now it's still excluded, but maybe that's because the field is private. It might be better to add comments saying why the fields are public.

Reply:
Good point! I have reverted the changes for this test class.